oneapp-server/types/Location.js

```diff
@@ -4,33 +4,51 @@ const { allow } = require('graphql-shield');
 const typeDef = gql`
   extend type Query {
     "All counties"
-    counties: [Location!]
+    counties: [County!]
     "Check if zipcode and county provided is a valid NJ zipcode"
     isValidNJZipcode(ZIPCODE: String!, COUNTY_NUMBER: Int!, IS_HOMELESS: Boolean!): Boolean
     "Returns county name of given user"
-    getCountyDetails: Location
+    getCountyDetails: County
+    "Check if valid USPS address"
+    isValidAddress(input: AddressInput!): Boolean
   }
 
-  type Location {
+  type County {
     COUNTY_NAME: String,
     COUNTY_NUMBER: Int,
   }
+
+  input CountyInput {
+    COUNTY_NAME: String,
+    COUNTY_NUMBER: Int,
+  }
+
+  input AddressInput {
+    ADDRESS: String,
```

Comment:
Address objects to match the legacy system should have `ADDRESS1` and `ADDRESS2` fields. Technically, however, for the USPS API, only `ADDRESS` (which the front-end should pull from `ADDRESS1`) and `ZIP` are used for validation.